apps/blockchain/lib/blockchain/ethash.ex

```diff
@@ -233,7 +232,10 @@ defmodule Blockchain.Ethash do
       |> initialize_mix(index, cache_size)
       |> binary_into_uint32_list()
 
-    uint32_cache = Enum.map(cache, &binary_into_uint32_list/1)
+    uint32_cache =
```

Comment:
This might also be a good place for `Enum.into/2` which does similar reduction logic

Reply:
Yeah, we had it as, 

```elixir
Enum.map(cache, fn {i, element} -> {i, binary_into_unint32_list(element)} end)
|> Enum.into(%{})
```

but we ended up going with this to reduce iterations through the cache. That being said, with your comment, I just took a look at `Enum.into/3` which is exactly what I think we need. So I'll update that.

Reply:
Updated in 2fb5acfcb8b0e96d75662e12e880719bd1a69fd8. It's not only more legible but seems to be faster too! Great call 👍